Source/GameBaseFramework/Private/Camera/Modifiers/GBFCameraModifierSpringArmTargetOffsetFromVelocity.cpp

```diff
@@ -0,0 +1,104 @@
+#include "Camera/Modifiers/GBFCameraModifierSpringArmTargetOffsetFromVelocity.h"
+
+#include "Camera/GBFPlayerCameraManager.h"
+#include "Camera/Modifiers/GBFCameraModifierUtils.h"
+
+#include <DrawDebugHelpers.h>
+#include <Engine/Canvas.h>
+#include <GameFramework/SpringArmComponent.h>
+
+UGBFCameraModifierSpringArmTargetOffsetFromVelocity::UGBFCameraModifierSpringArmTargetOffsetFromVelocity() :
+    OffsetScale( 1.0f, 1.0f, 0.0f ),
+    Operation( EGBFCameraModifierAttributeOperation::Override ),
+    bUseOffsetCurve( false ),
+    OffsetInterpolationSpeed( 2.0f ),
+    bUseInterpolationSpeedCurve( false ),
+    bUseYawDifferenceCurve( 0 ),
+    ViewTargetVelocity( 0.0f ),
+    OffsetCurveValue( 0.0f ),
+    InterpolationSpeed( 0.0f ),
+    RotationDifference( 0.0f ),
+    RotationDifferenceMultiplier( 1.0f )
+{
+}
+
+void UGBFCameraModifierSpringArmTargetOffsetFromVelocity::ModifyCamera( float delta_time, FVector view_location, FRotator view_rotation, float fov, FVector & new_view_location, FRotator & new_view_rotation, float & new_fov )
```

Comment:
const

Reply:
several parameters (view_location, new_view_location, new_view_rotation, new_fov) seems to be unused

Reply:
done

Reply:
done